SpaceEnvironment_TotalLaunches/js/area.js

```diff
@@ -0,0 +1,82 @@
+$(function() {
+  $('#hcContainer').highcharts({
+    // Load Data in from Google Sheets
+    data: {
+      googleSpreadsheetKey: '1FpDY2RRA1EW5Pk7cK9tRSstaH6P0f-2GEDc_VjNRI4o',
+      googleSpreadsheetWorksheet: 1,
+      complete: function (data) {
+        // Display legend alphabetically
+        let dataItems = data.series
+        let dataNames = []
+
+          dataItems.forEach(item => {
+              item.legendIndex = ''
+              dataNames.push(item.name)
+            })
+
+            notSorting = dataNames.filter(arrayItem => arrayItem === 'Others')
+            alphaSorting = dataNames.filter(arrayItem => arrayItem !== 'Others')
+            alphaSorting.sort().push(notSorting[0])
```

Comment:
You don't need to change this for this particular implementation, but if you were to future-proof this code, you would want to check to make sure that notSorting contained values, otherwise this could result in an error as `notSorting[0]` would be undefined.